code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm

```diff
@@ -93,6 +124,16 @@
 	result = /obj/item/reagent_containers/food/snacks/spidereggsham
 	subcategory = CAT_MEAT
 
+/datum/crafting_recipe/food/sushie_egg
```

Comment:
Is this a typo?

Reply:
E is at the end for labeling. Its a active chose for later ideas and improvements

Reply:
that doesn't make sense